src/librustdoc/passes/mod.rs

```diff
@@ -131,23 +131,16 @@ impl ConditionalPass {
 crate enum DefaultPassOption {
     Default,
     Coverage,
-    None,
 }
```

Comment:
In a later PR, I feel like we should get rid of this enum altogether. It doesn't need to happen though.